app/views/govuk_component/document_footer.raw.html.erb

```diff
@@ -29,17 +29,38 @@
         <p><%= title %>: <span class="other-date definition"><%= date %></span></p>
       <% end %>
     <% end %>
-    <% if history.any? %>
+    <%
```

Comment:
This could be moved up to the top of the template, with the other argument defaults (and probably use the same default assignment style)